ui/src/components/tree/QTree.js

```diff
@@ -329,8 +329,7 @@ export default defineComponent({
             lazy.value[ key ] = 'loaded'
             node[ props.childrenKey ] = Array.isArray(children) === true ? children : []
             nextTick(() => {
-              const localMeta = m.value[ key ]
-              if (localMeta && localMeta.isParent === true) {
+              if (m && m.isParent === true) {
```

Comment:
For some reason `isParent` doesn't get recalculated after setting node's children.

Can someone help me with that, sadly I don't really understand what's the problem here.

Reply:
My bad, fixed now.